action.yml

```
@@ -59,6 +63,8 @@ runs:
   - ${{ inputs.version-keys }}
   - '--case-sensitive-keys'
   - ${{ inputs.case-sensitive-keys }}
+  - 'trim-start-from-branch'
```

Add 2 hyphens to the beginning of the input name

```suggestion
  - '--trim-start-from-branch'
```